tendrl/notifier/notification/handlers/mail_handler.py

```diff
@@ -58,9 +60,13 @@ def get_config_help(self):
         return config_help
 
     def set_destinations(self):
-        # TODO(anmolbabu):User configuration will not just be destination email
-        # but it would also include user level subscriptions the capability
-        # underneath needs to be enhanced for that.
+        # TODO(gowtham):Get user ids from indexes who enabled
+        # email notification
+        pass
+
+    def get_alert_destinations(self):
+        # TODO(gowtham): get user emailds only who have enabled
+        # email notfication
```

Comment:
This function was intended to get list of email ids that are to be notified of the current alert being processed which is why alert as a parameter was passed. I think you would need it when you enable a feature where users can even subscribe only to a list of alert types that is a subset of all supported alert types.